piparty.py

```diff
@@ -16,358 +22,377 @@
+                        move_color = Colors.Blue40.value
                     else:
-                        move.set_leds(*colors.Colors.Yellow.value)
+                        move_color = Colors.Yellow.value
 
-                elif game_mode == common.Games.Zombies:
-                        move.set_leds(*colors.Colors.Zombie.value)
+                # Set everyone to Zombie color (probably green)
+                elif game_mode == Games.Zombies:
+                    move_color = Colors.Zombie.value
 
-                elif game_mode == common.Games.Commander:
+                # Set even moves to red and odds to blue
+                elif game_mode == Games.Commander:
                     if move_num % 2 == 0:
-                        move.set_leds(*colors.Colors.Orange.value)
+                        move_color = Colors.Red.value
                     else:
-                        move.set_leds(*colors.Colors.Blue.value)
+                        move_color = Colors.Blue.value
 
-                elif game_mode == common.Games.Swapper:
+                # Set colors to bounce between Magenta and Green
+                elif game_mode == Games.Swapper:
                     if (time.time()/5 + random_color)%1 > 0.5:
-                        move.set_leds(*colors.Colors.Magenta.value)
+                        move_color = Colors.Magenta.value
                     else:
+                        move_color = Colors.Green.value
 
-                        move.set_leds(*colors.Colors.Green.value)
-
-                elif game_mode == common.Games.FightClub:
-                        move.set_leds(*colors.Colors.Green80.value)
+                # Set all moves to green
+                elif game_mode == Games.FightClub:
+                    move_color = Colors.Green80.value
 
-                elif game_mode == common.Games.NonStop:
-                        move.set_leds(*colors.Colors.Turquoise.value)
+                # Set all moves to turquoise
+                elif game_mode == Games.NonStop:
+                    move_color = Colors.Turquoise.value
 
-                elif game_mode == common.Games.Tournament:
+                # Set moves to random colors
+                elif game_mode == Games.Tournament:
                     if move_num <= 0:
                         color = time.time()/10%1
                         color = colors.hsv2rgb(color, 1, 1)
-                        move.set_leds(*color)
+                        move_color = color
                     else:
-                        move.set_leds(*colors.Colors.Blue40.value)
+                        move_color = Colors.Blue40.value
 
-
-                elif game_mode == common.Games.Ninja:
-                    if move_num <= 0:
-                        move.set_leds(random.randrange(100, 200),0,0)
+                # Set moves to random colors
+                elif game_mode == Games.Ninja:
+                    if move_num <= 0: # TODO - How does this happen?
```

Comment:
I believe this is just for the first controller (move 0) to be a different random color. i.e. one controller is a random value, the rest are red for this game mode.